core/src/main/java/org/commonjava/maven/ext/core/impl/RelocationManipulator.java

```diff
@@ -128,13 +161,66 @@ private boolean apply( final Project project, final Model model ) throws Manipul
             result |= updateDependencies( relocations, project.getAllResolvedProfileDependencies( session ).get( profile ) );
 
         }
+
+        result |= updateAllPlugins( model, project, relocations );
+
+        return result;
+    }
+
+    private boolean updateAllPlugins( final Model model, final Project project, final WildcardMap<ProjectVersionRef> relocations ) throws ManipulationException
+    {
+        Map<String, Plugin> pluginMap = getManagedPluginMap( model );
+        List<PluginReference> refs =  findPluginReferences( model, project, pluginMap );
+        boolean result = updatePlugins( relocations,  refs );
+
+        pluginMap = getPluginMap( model );
```

Comment:
As per above use the Project API - it also resolves any variables for you.

Reply:
OK, it is agreed to make this change.